ghost/core/package.json

```
@@ -243,10 +243,11 @@
   },
   "optionalDependencies": {
     "@tryghost/html-to-mobiledoc": "3.3.3",
-    "sqlite3": "5.1.7"
+    "better-sqlite3": "catalog:"
   },
   "devDependencies": {
     "@actions/core": "3.0.1",
+    "@eslint/js": "catalog:",
```

Biome (2.5.0)

[error] The key `@eslint/js` was already declared.

(lint/suspicious/noDuplicateObjectKeys)